server/core/src/main/java/org/infinispan/server/core/transport/NettyTransport.java

```diff
@@ -340,10 +354,18 @@ public void updateTotalBytesRead(int bytes) {
       return channel;
    }
 
-   private EventLoopGroup buildEventLoop(int nThreads, DefaultThreadFactory threadFactory) {
-      EventLoopGroup eventLoop = EPollAvailable.USE_NATIVE_EPOLL ? new EpollEventLoopGroup(nThreads, threadFactory) :
+   public static MultithreadEventLoopGroup buildEventLoop(int nThreads, ThreadFactory threadFactory,
+         String configuration) {
+      MultithreadEventLoopGroup eventLoop = EPollAvailable.USE_NATIVE_EPOLL ? new EpollEventLoopGroup(nThreads, threadFactory) :
             new NioEventLoopGroup(nThreads, threadFactory);
-      log.createdNettyEventLoop(eventLoop.getClass().getName(), configuration.toString());
+      log.createdNettyEventLoop(eventLoop.getClass().getName(), configuration);
+      return eventLoop;
+   }
+
+   public static MultithreadEventLoopGroup buildEventLoop(int nThreads, Executor executor, String configuration) {
```

Comment:
I think this method is not used anymore.

Reply:
Thought I had removed that, let me do that